index.js

```diff
@@ -67,6 +67,44 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(e) {
+    toggleColor(header)
+})
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector('button.like-button')
+const raffyLikes = document.querySelector('p.likes')
+
+likeButton.addEventListener('click', function() {
+    traveler.likes += 1
```

Comment:
Remember that we don't want our likes to persist! You're on the right track with: 

- First find the like button element on the page ✅ 
- Then add an event listener to the like button  ✅ 

Then what if I want to update the number of likes that's stored within the `p` tag only? I'd first need to extract that number from the p tag, and then what?